resources/partial/notification-type-springboot/${id}-notification-type.client-extension-config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "com.liferay.notification.internal.configuration.ClientExtensionNotificationTypeConfiguration~${id}" : {
```

Comment:
Has this type already been review and merge into `liferay-portal/master` or is it just a PoC so far?

Reply:
ok, then let's keep working on the PR and only merge it once the configuration is finalized and at least lives in a nightly docker build of DXP :)

Reply:
Ok!